cli/src/commands/add/tailwind.ts

```
@@ -0,0 +1,396 @@
+														const text = element.getText();
+														return text.startsWith('"') || text.startsWith("'")
+															? text.slice(1, -1)
+															: text;
+													});
+											}
+										} else if (
+											initializer.getKind() === SyntaxKind.SpreadElement
+										) {
+											// Skip spread elements as they're not valid in JSON
+											continue;
+										} else if (
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)